src/frontend/sharedComponents/BottomSheetModal/Backdrop.tsx

```diff
@@ -9,25 +9,24 @@ export const Backdrop = ({
   style,
   ...rest
 }: BottomSheetBackdropProps) => {
-  const animatedOpacity = React.useMemo(
-    () =>
-      interpolate(animatedIndex, {
-        inputRange: [0, 1],
-        outputRange: [0, 0.3],
-        extrapolate: Extrapolate.CLAMP,
-      }),
-    [animatedIndex]
+  //Use memo was causing a flickering effect when the backdrop was opened.
```

Comment:
interesting. slightly worried that removing this might break something else (i think i took this from the library's example), so worth testing other components that use this if possible

Reply:
The implementation is an adapted version of this example for animating the backdrop background: https://github.com/gorhom/react-native-bottom-sheet/blob/v2/example/src/components/customBackground/CustomBackground.tsx

i have a feeling that the changes here might break that but if you can verify that it doesn't then works for me